src/js/jsx/sections/style/Type.jsx

```diff
@@ -548,23 +550,43 @@ define(function (require, exports, module) {
                 weightListID = "weights-" + this.props.document.id;
 
             return (
-                <div ref="type" className="type sub-section">
-                    <header className="sub-header">
-                        <h3>
-                            {strings.STYLE.TYPE.TITLE}
-                        </h3>
-                        <Gutter />
-                        <hr className="sub-header-rule"/>
-                        <div className="button-cluster">
+                <div ref="type">
+                    <div className="formline">
+                        <div className="control-group__vertical">
+                            <ColorInput
+                                id={colorPickerID}
+                                ref="color"
+                                className="type"
+                                context={collection.pluck(this.props.document.layers.selected, "id")}
+                                title={strings.TOOLTIPS.SET_TYPE_COLOR}
+                                editable={!this.props.disabled}
+                                defaultValue={colors}
+                                opaque={this.state.opaque}
+                                onChange={this._handleColorChange}
+                                onColorChange={this._handleOpaqueColorChange}
+                                onAlphaChange={this._handleAlphaChange}
+                                swatchOverlay={typeOverlay} />
                         </div>
-                    </header>
-
+                        <div className="control-group__vertical control-group__no-label">
+                            <BlendMode
+                                    {...this.props}
```

Comment:
BlendMode has a limited props, so it might not be the best scenarios for using props transfer here.